lib/smart_app_launch/smart_stu2_suite.rb

```diff
@@ -32,6 +39,21 @@ class SMARTSTU2Suite < Inferno::TestSuite
       post_authorization_uri: "#{Inferno::Application['base_url']}/custom/smart_stu2/post_auth"
     }
 
+    description <<~DESCRIPTION
+      The SMART App Launch Test Suite verifies that servers correctly implement
+      the [SMART App Launch IG](http://hl7.org/fhir/smart-app-launch/STU2/).
+      To get started, please first register the Inferno client as a SMART App
+      with the following information:
+
+      * SMART Launch URI: `#{config.options[:launch_uri]}`
+      * OAuth Redirect URI: `#{config.options[:redirect_uri]}`
+
+      If using asymmetric client authentication, register Inferno with the
+      following JWK Set Url:
+
+      * `#{Inferno::Application[:base_url]}/custom/smart_stu2/.well-known/jwks.json`
+    DESCRIPTION
+
```

Comment:
Should we add something here like:

```md
Asymmetric client authentication requires a JWK Set URL served by Inferno to be accessible by the system under test.  If this is not possible, you may register Inferno using the [ONC-hosted JWK Set URL](https://inferno.healthit.gov/custom/smart_stu2/.well-known/jwks.json).
```
Downside of this is that this will be redudant for the 'regular' case of running these tests on inferno.healthit.gov.

Reply:
Also, I think we should add this blurb to the 'input instructions' of each as we do in g10 (though standalone wouldn't need the SMART Launch URI) from a usability of this perspective.